config/routes.rb

```diff
@@ -19,19 +18,14 @@
     get '/download-csv', to: 'download#download_csv', as: 'download_csv'
   end
 
-  resources :api_users, path: 'create-api-key'
-  get '/api_users/new', to: redirect('/create-api-key', status: 301)
-
   get '/registers-in-progress', to: 'registers#in_progress'
 
-  resources :suggest_registers, path: 'suggest-register', except: %i[show edit]
-  get 'suggest-register/complete', to: 'suggest_registers#complete'
+  resources :api_users, path: 'create-api-key'
 
-  resources :request_registers, path: 'request-register', except: %i[show edit]
-  get 'request-register/complete', to: 'request_registers#complete'
+  get '404', to: 'errors#not_found'
```

Comment:
can we move `404` and `500` to the bottom with the redirects as I think they make more sense there.

Reply:
Compromised and put them above `/admin` routes.